web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -193,12 +200,23 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
       if (!alreadyExists) {
         handleOpen();
         const res = await createReviewPeriod(periodToAdd, csrf);
-        const data =
-          res && res.payload && res.payload.data ? res.payload.data : null;
-        data && dispatch({ type: ADD_REVIEW_PERIOD, payload: data });
+        const data = res?.payload?.data ?? null;
+        console.log('Data');
```

Comment:
I imagine you are finished with these two `console.log` calls.

Reply:
Removed